frontend_app_workspace_advanced/src/container/WorkspaceAdvanced.jsx

```diff
@@ -183,10 +191,34 @@ export class WorkspaceAdvanced extends React.Component {
     }))
   }
 
+  handleSubscriptionCreated = data => {
+    if (this.state.subscriptionsRequests.some(request =>
+      request.author.user_id === data.fields.subscription.author.user_id
+    )) return
+
+    this.setState(prev => ({
+      subscriptionsRequests: [
+        ...prev.subscriptionsRequests,
+        data.fields.subscription
+      ]
+    }))
+  }
+
+  handleSubscriptionModified = data => {
+    this.setState(prev => ({
+      subscriptionsRequests: prev.subscriptionsRequests.map(request =>
```

Comment:
I would also call this state subscriptionRequestList since it is an array

Reply:
Updated